core/validation/initial_check.py

```diff
@@ -0,0 +1,150 @@
+"""Initial Pre-transformation Validation
+
+Provides functionality for validating a workbook before transformations occur.
+"""
+
+import pandas as pd
+
+import core.validation.failures as vf
+from core.const import REPORTING_PERIOD_DICT
+
+
+def extract_round_three_submission_details(workbook: dict[str, pd.DataFrame]) -> dict[str, str]:
+    """
+    Extract submission details for round three from the given workbook.
+
+    :param workbook: A dictionary where keys are sheet names and values are pandas DataFrames.
+    :return: A dictionary containing the extracted submission details.
+    """
+    details_dict = {}
+
+    sheet_a1 = workbook.get("1 - Start Here")
+    if sheet_a1 is not None:
+        details_dict["Form Version"] = sheet_a1.iloc[6][1]
+        details_dict["Reporting Period"] = sheet_a1.iloc[4][1]
+        details_dict["A1"] = "valid"
+    else:
+        details_dict["Form Version"] = None
```

Comment:
this feels unnecessary. First of all it's probably better not to call the else at all, and just access the expected keys with a get method. Then missing data is implied by it's absence, not entered into a dict as None values (this is better practice in general). 

But in this case we shouldn't worry about that, instead I think it'd make more sense to add a check before these that checks if these sheets exist, if not immediately cease validating and return an error (since the other code will not find anything if all is missing).